compose-app/src/main/java/com/mapbox/maps/extension/compose/MapPreviewPlaceHolder.kt

```diff
@@ -0,0 +1,41 @@
+package com.mapbox.maps.extension.compose
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.unit.dp
+
+
+@Composable
+internal fun MapboxMapPreviewPlaceHolder(
```

Comment:
should we give access to this place holder? maybe user want to set some custom image?

Reply:
yep, we can support such use case, however it would be hard to expose the composable function, maybe we can abstract the state out of the placeholder and let user to set it when using MapboxMap composable function.
Any idea how we can expose it?